### PR TITLE
brillo: new package at 1.4.9

### PR DESCRIPTION
**Pull request by mmatongo**
## Description of package

Control the brightness of backlight and keyboard LED devices
## Installed manifest of package

```
/var/db/kiss/installed/brillo/version
/var/db/kiss/installed/brillo/sources
/var/db/kiss/installed/brillo/manifest
/var/db/kiss/installed/brillo/depends
/var/db/kiss/installed/brillo/checksums
/var/db/kiss/installed/brillo/build
/var/db/kiss/installed/brillo/
/var/db/kiss/installed/
/var/db/kiss/
/var/db/
/var/
/usr/lib/udev/rules.d/92-com.gitlab.CameronNemo.brillo.rules
/usr/lib/udev/rules.d/
/usr/lib/udev/
/usr/lib/
/usr/bin/brillo
/usr/bin/
/usr/
```

## New package

- [Y] Latest upstream version.
- [Y] I agree to maintain this package.
- [Y] I agree to notify @dylanaraps if I can no longer maintain this package.

### CODE REVIEW

File: community/brillo/depends [hunk 1]
```diff
@@ -0,0 +1,2 @@
+go
+go-md2man
```
Comment (dylanaraps) [inline community/brillo/depends:2]: Is this needed solely for docs?

Comment (dylanaraps) [reply community/brillo/depends:2]: Seems to be the case. I'd much rather we disable docs or ship a pre-generated manual page than add a dependency on Go to build something written in C.

Comment (mmatongo) [reply community/brillo/depends]: Seems `go-md2man` is quite useless, it serves no purpose other than bloat as a dependancy for `brillo`.
I've pushed a new commit removing it.

File: community/brillo/depends [hunk 2]
```diff
@@ -0,0 +1 @@
+go
```
Comment (dylanaraps) [inline community/brillo/depends:1]: Is Go still a dependency?

Comment (mmatongo) [reply community/brillo/depends:1]: Only as a `make` dependancy. With regards to `go-md2man`  lines 6, lines 70-76 eliminate the need it.

Comment (mmatongo) [reply community/brillo/depends]: I have pushed the new commits and updated the manifest to reflect the new changes made.

File: community/brillo/build [hunk 3]
```diff
@@ -0,0 +1,6 @@
+#!/bin/sh -e
+
+sed -i '6d;70,76d' Makefile
+
+make
+make DESTDIR="$1" install
```
Comment (dylanaraps) [inline community/brillo/build:6]:
```sh
#!/bin/sh -e

make
make DESTDIR="$1" install.bin install.udev
```

File: community/brillo/depends [hunk 4]
```diff
@@ -0,0 +1 @@
+go make
```
Comment (dylanaraps) [inline community/brillo/depends:1]: The depends file can be removed. With the build file changes no dependencies are needed.

File: community/brillo/build (hunk 3 above)
Comment (mmatongo) [reply community/brillo/build:6]: `sed -i '6d;70,76d' Makefile` removes go-md2man as a dependency.

Comment (mmatongo) [reply community/brillo/build]: Nice

File: community/brillo/depends (hunk 4 above)
Comment (mmatongo) [reply community/brillo/depends]: dependency removed and new commits pushed.

**Pull request merged**